code/game/objects/items/defibrillator.dm

```diff
@@ -165,15 +171,15 @@
 	var/defib_heal_amt = damage_threshold
 
 	//job knowledge requirement
-	var/skill = user.skills.getRating(SKILL_MEDICAL)
-	if(skill < SKILL_MEDICAL_PRACTICED)
+	var/medical_skill = user.skills.getRating(SKILL_MEDICAL)
+	if(medical_skill < SKILL_MEDICAL_PRACTICED)
 		user.visible_message(span_notice("[user] fumbles around figuring out how to use [src]."),
 		span_notice("You fumble around figuring out how to use [src]."))
-		var/fumbling_time = SKILL_TASK_AVERAGE - (SKILL_TASK_VERY_EASY * skill) // 3 seconds with medical skill, 5 without
+		var/fumbling_time = SKILL_TASK_AVERAGE - (SKILL_TASK_VERY_EASY * medical_skill) // 3 seconds with medical medical_skill, 5 without
 		if(!do_after(user, fumbling_time, NONE, H, BUSY_ICON_UNSKILLED))
 			return
-	else
-		defib_heal_amt *= skill * 0.5 //more healing power when used by a doctor (this means non-trained don't heal)
+	else // for ye whom may venture here: if you kill this "else", kill the special snowflake bit in check_revive()
```

Comment:
the fact you're putting this note here should tell you that what you're doing is wrong.
Make the define actually correct and don't gate it behind an else at all, then you don't have to worry about someone breaking health scanners in the future

Reply:
qv said removing the else is balance last time i tried this, but sure

Reply:
You have to change the define, not just remove the else